src/test/regress/multi_schedule

```diff
@@ -33,6 +33,7 @@ test: multi_load_data
 test: multi_insert_select_behavioral_analytics_create_table
 test: multi_insert_select_behavioral_analytics_basics multi_insert_select_behavioral_analytics_single_shard_queries multi_insert_select_non_pushable_queries
 test: multi_insert_select
+test: multi_cross_shard
```

Comment:
Any chance you could find a parallel group to fit this in?  Without aggressively looking for that our test times will grow way too quickly.